sdk/maps/azure-maps-render/swagger/README.md

```diff
@@ -0,0 +1,43 @@
+# Azure Maps Render for Python
+
+> see <https://aka.ms/autorest>
+
+## Setup
+
+Install Autorest v3
+
+```ps
+npm install -g autorest
+```
+
+## Generation
+
+```ps
+cd <swagger-folder>
+autorest --v3 --python
+```
+
+## Settings
+
+```yaml
+tag: '2022-08-01'
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/48bb51ee0753ed56a88b3e7f989a70bf19ba96bb/specification/maps/data-plane/Render/readme.md
+output-folder: ../azure/maps/render/_generated
+namespace: azure.maps.render
+package-name: azure-maps-render
+no-namespace-folders: true
+use-extension:
+  "@autorest/modelerfour": "4.22.3"
```

Comment:
Can we try removing this extension dependency here? In C# we can use 4.23.0+ if the render API version is 2022-08-01